benchmarks/mongo-es-speed/buildState.js

```diff
@@ -27,12 +29,18 @@ function reporterHandler() {
 
 setTimeout(reporterHandler, 500);
 
-store.loadEventsByTypes(TYPES, () => (eventCounter++)).then(() =>
-    // eslint-disable-next-line no-console
-    console.log(DONE_TOKEN, JSON.stringify({
-        memory: process.memoryUsage()
-    }))
-).catch(err =>
-    // eslint-disable-next-line no-console
-    console.log(ERR_TOKEN, err)
-);
+store
+    .loadEventsByTypes(TYPES, () => eventCounter++)
```

Comment:
Stairs-alike code. Function `loadEventsByTypes` and subsequent `then` and `catch` calls is not uniform. Function `then` is attached to `store.loadEventsByTypes` and has no relation to pure `store` object